### PR TITLE
[DEL] MRP: Delete Alternative Work Centers

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR deletes the "[Using an alternative Work Center](https://www.odoo.com/documentation/14.0/applications/inventory_and_mrp/manufacturing/management/alternative_wc.html)" doc in the "Manufacturing management" folder of the Manufacturing app documentation for version 14.0. The information in this doc has been rewritten and expanded upon in the "[Manage work orders using work centers](https://www.odoo.com/documentation/14.0/applications/inventory_and_mrp/manufacturing/management/using_work_centers.html)" doc, located in the same folder. Any references to the deleted doc in the documentation file tree have been removed. A redirect from the deleted doc to the new doc has been added as well.

Two additional PRs will be pushed to implement the same change for versions 15.0 and 16.0.

### CODE REVIEW

**Comment by hojo-odoo** [comment]
Hey @StraubCreative,

This is another deletion PR. I've already added the redirect this time.

Thanks,
hojo

**Review by StraubCreative — APPROVED** [review]
LGTM 👍 
@samueljlieber can you give me a sanity check here and merge **only for 14.0** if all g, thanks.

@robodoo delegate=samueljlieber

File: redirects/14.0.txt [hunk 1]
```diff
@@ -54,6 +54,7 @@ applications/inventory_and_mrp/inventory/management/incoming/three_steps.rst app
 applications/inventory_and_mrp/inventory/management/incoming/two_steps.rst applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst                         # /inventory/management/incoming/* --> /management/shipments_deliveries/*
 applications/inventory_and_mrp/inventory/management/replenishment_strategies/virtual_warehouses.rst applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
 applications/inventory_and_mrp/inventory/routes/concepts/inter_warehouse.rst applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
+applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst
```
Comment (samueljlieber) [inline redirects/14.0.txt:58]: This redirect is for the manufacturing app scope, but was placed in the inventory scope. Add a new section for manufacturing below 🙂
```suggestion

# applications/manufacturing

applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst

```

**Review by samueljlieber — APPROVED** [review]
Nice job @hojo-odoo! Looks good except the redirect is in the Inventory section, need to move it to a Manufacturing section. Other than that we are good to merge for only 14.0. I will push up the change now 🙂

**Comment by samueljlieber** [comment]
@robodoo retry

**Pull request closed without merging**